scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -324,6 +324,7 @@ impl ReplicatorConsumer {
                 })
                 .join(",");
 
+            let values_for_update = &mut values_for_update.to_vec();
             values_for_update.remove(1);
```

Comment:
Tip: `run_statement` accepts an immutable slice, so no need to physically pop the value, you can just shorten the slice and no cloning will be needed:

```rust
let values_for_update = &values_for_update[..values_for_update.len()-1];
```

Reply:
Oh, wait, `.remove(1)` removes the first element, not one element from the end. It's OK, then - my bad.